boards/intel/niosv_g/Kconfig.niosv_g

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2023-2024, Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NIOSV_G
+	bool
```

Comment:
Please remove bool, same reason as given in other pr.
```suggestion
```

Reply:
removed